lightning/mydump/parser.go

```diff
@@ -154,12 +155,51 @@ const (
 	tokFalse
 	tokHexString
 	tokBinString
+	tokInteger
 	tokSingleQuoted
 	tokDoubleQuoted
 	tokBackQuoted
 	tokUnquoted
 )
 
+// String implements the fmt.Stringer interface
+//
+// Mainly used for debugging a token.
+func (t token) String() string {
```

Comment:
I think the better way to implement the `fmt.Stringer` here is using a string array because of the token enum is an autoincrement number. For example,
```go
var tokStringify = [...] {
	tokNil: "<Nil>",
	tokRowBegin: "RowBegin",
	// ...
}
func (t token) String() string {
	if t < tokNil || t > tokUnquoted || tokStringify[t] == "" {
		return fmt.Sprintf("unknown(%d), t)
	}
	return tokStringify[t]
}
```